CONTRIBUTING.md

```diff
@@ -129,10 +129,33 @@ To start che-theia in `dev-mode` with yarn (not using the production che-theia g
 Running dev theia would be located in the che-dev container and `theia-dev-flow` endpoint:
 ![Che-Theia-dev-endpoint](https://raw.githubusercontent.com/eclipse/che-theia/assets/theia-dev-flow-endpoint.png)
 
-
-
 ### Just want to build the plugin and run with the existing Theia image
 If you do not have any changes on Theia or Che-theia extension, you could just build the plugins with `build ... containers-plugin` or `build ... factory-plugin`
 and run these plugins with the existing che-theia app:
 `run ... HOSTED che-theia + container-plugin` or `run ... HOSTED che-theia + factory-plugin`
 
+## Contributing to other parts of Che Theia
```

Comment:
Now that we have more contents on this file, could you add a TOC at the beginning of this file.

Reply:
@sunix done